wordpress/wp-content/plugins/cds-base/classes/Modules/Users/Users.php

```diff
@@ -163,6 +169,33 @@ public function addToBlog($uId, $role)
         }
     }
 
+    // note this is only available to super admins
+    // for use when setting up a new site / user
+    public function sendWelcome($uId, $email, $userExists)
+    {
+        if (!$userExists) {
+            $userInfo = get_userdata($uId);
+            $unique = get_password_reset_key($userInfo);
+            $uniqueUrl = network_site_url(
+                "wp-login.php?action=rp&key=$unique&login=" . rawurlencode($userInfo->user_login),
+                'login'
+            );
+        }
+
+        // phpcs:disable
+        $subject = __("Your GC Articles site is ready", "cds-snc");
+        $message = __('@todo add welcome information here', "cds-snc") . "\r\n\r\n";
+        $message .= __('To set your GC Articles account password, please visit the following address:', "cds-snc") . "\r\n\r\n";
+        
+        if(!$userExists){
+            // send reset pass url
+            $message .= $uniqueUrl;
+        }
+        // phpcs:enable
```

Comment:
If the user exists already, there's no link, eh? Should we link them to the login page for the new site?

Reply:
Or does the existing message cover it? (eg, maybe we don't need a welcome message unless it's a totally new user?)

Reply:
Was thinking about a case where this could be the 2nd or 3rd site the person has access to.  In this case we don't want to them to reset the password.

In which case I should remove this as well.

> $message .= __('To set your GC Articles account password, please visit the following address:', "cds-snc") . "\r\n\r\n";